bin/coreos-install

```diff
@@ -26,6 +26,10 @@ fi
 : ${VERSION_ID:=current}
 CHANNEL_ID=${GROUP:-stable}
 
+BOARD=$(gawk --field-separator '=' '/COREOS_RELEASE_BOARD=/ { print $2 }' /usr/share/coreos/release)
```

Comment:
this breaks compatibility with installing coreos from non-coreos system, i.e. a debian rescue system

```bash
root@rescue:~# ./coreos-install -d /dev/sda -C stable -c ./cloud-config.yaml
gawk: fatal: cannot open file `/usr/share/coreos/release' for reading (No such file or directory)
root@rescue:~#
```